docs/MapColonies/vector/Services/geocoding/README.md

```diff
@@ -0,0 +1,1024 @@
+---
+id: vector-geocoding-service
+slug: info
+title: Geocoding 📍
+description: Search engine for locations and Control Language
+tags:
+  - Vector
+  - API
+  - Geocoding
+---
+
+# Geocoding 📍
+
+:::info
+Click [here](/docs/MapColonies/vector/Services/geocoding/api) for the OpenAPI
+:::
+
+## TL;DR
+
+Search for places, control tiles, routes and items. Ability to convert from WGS 84 Coordinates to Control Tile/ US Army MGRS. 
+
+## Terminology
+
+### What is the Control layer?
+
+:::tip
+Check [`Control Search request examples`](#control-examples).
+:::
+
+MapColonies has its own Control Reference System. Like the US Army MGRS, we divided our user's area of interest to Tiles. Each tile is 10kmX10km and has Sub-Tiles which are 1kmX1km. Each tile has 100 sub-tiles. <br/>
+A tile's name is exactly 3-letters, while a Sub-Tile is a 2-digit number. <br/>
+
+
+<details>
```

Comment:
Maybe add a short explanation for these responses, or in my opinion just write examples as plain text because we have examples at the end of this document.